### ISSUE TITLE
Discovery of json/yml of "fr" family doesn't work

### ISSUE THREAD

**Issue by DiesIrae** [vazco/meteor-universe-i18n#65]
Hi,

When I put a file "fr.i18n.yml" into my translation folder, then set the locale to "fr", French language is not loaded. But when I do the same with other locales, as "es", "de", "ru"... it works!

This problem seems to be on all fr-XX family, and is the same with json. I temporarily solved it adding `UNIVERSE_I18N_LOCALES = all`.

**Comment by cristo-rabani** [comment]
Hi, 
Thanks for reporting.
Our team should check this soon

**Comment by ibussieres** [comment]
Any update on this, having the same issue here.

**Comment by janowsiany** [comment]
Hey, @DiesIrae and @ibussieres 
could you please provide some kind of reproduction? I tried to do this but for me it seems to work. The tests i am running:
```js
it('should load fr translations also', async () => {
    expect(i18n.isLoaded('fr')).to.not.be.true;
    await i18n.setLocale('fr');
    expect(i18n.isLoaded('fr')).to.be.true;
});
```
If you could also specify your environment, which meteor version are you running on?

**Comment by cristo-rabani** [comment]
Hello @ibussieres @DiesIrae 

Any details?
Path to file of your french localization with some basic content will be helpful.

**Comment by ibussieres** [comment]
`imports/locales/fr.18n.yml`
`imports/locales/en.i18n.yml`

English is working fine.

Basic content I just tested, still not working : 

```User:
  email: Courriel
  password: Mot de passe```

**Comment by janowsiany** [comment]
@ibussieres I am wondering if it is just a typo but you are missing `i` in your french translations yml file.
It should be:
`imports/locales/fr.i18n.yml`
Could you please check it and confirm weather it was an issue or just a typo in this comment?

**Comment by ibussieres** [comment]
Yes, sorry. Just a typo.

![image](https://cloud.githubusercontent.com/assets/422296/26529185/69925ba2-4389-11e7-9a69-38f126af2900.png)

**Comment by DiesIrae** [comment]
Hi,

On my side I couldn't reproduce the problem. Sorry!

**Comment by ibussieres** [comment]
Even using my sample locale file ?

**Comment by ibussieres** [comment]
New information. 

I simply renamed `en.i18n.yml` to `es.i18n.yml` and the getLanguages returned `[]`. I then renamed `fr.i18n.yml` to `en.i18n.yml` and getLanguages returned `["en"]` once more and all my French translations were immediately applied. It really seems as if the fr language family is not being detected.

As you can see, I see es6 imports. Maybe that is part of the issue or can at least help replicate the issue, I import the language files this way, in `client/main.jsx`.

```
import '../imports/locales/en.i18n.yml';
import '../imports/locales/fr.i18n.yml'; 
```

**Comment by dikatok** [comment]
Hi guys, thanks for the great module.

Recently I added a translation file to my app but it doesn't seem to correctly load the translations. The language is bahasa which is `id` or `id-ID` which is also mentioned in the readme. I have also tried to manually set the locale via chrome dev console but it didn't work. Is it related to the OP's problem?

Below is how I called the `setLocale` and the translation file.
![image](https://user-images.githubusercontent.com/11844490/29663396-13114124-88f5-11e7-8ea1-25d749e83206.png)
![image](https://user-images.githubusercontent.com/11844490/29663417-3716dc5a-88f5-11e7-91fc-8b46dfb004d3.png)
![image](https://user-images.githubusercontent.com/11844490/29663425-47e0b7d6-88f5-11e7-9094-c08b2ef60945.png)
